packages/core/src/Modules/Cashier/Components/payment-agent-details.jsx

```diff
@@ -15,7 +15,9 @@ class Detail extends React.Component {
                     href={`${action ? `${action}:` : ''}${value}`}
                     {...rest}
                 >
-                    {value}
+                    <Text styles={is_bold && { color: 'var(--brand-red-coral)' }} weight={is_bold && 'bold'} size='xs'>
```

Comment:
Did you observe the behavior when `is_bold` is false?

Reply:
Yes of course I did. On line 41 we use `Detail` without `is_bold`

Reply:
Do you mean the condition has a problem?
`is_bold && { color: 'var(--brand-red-coral)' }`
Is there a possible situation that we pass `is_bold = false` prop to `Detail` instead of ignoring it?

Reply:
Yea I mean what happens if we pass it being `false`?